.circleci/ci/it-tests.js

```diff
@@ -48,16 +49,38 @@ try {
     });
 
     // Run integration tests
-    ci.dir('it.tests', () => {
-        ci.sh(`mvn clean verify -U -B -Plocal,${classifier}`); // The -Plocal profile comes from the AEM archetype 
-    });
+    if (TYPE === 'integration') {
+        ci.dir('it.tests', () => {
+            ci.sh(`mvn clean verify -U -B -Plocal,${classifier}`); // The -Plocal profile comes from the AEM archetype 
+        });
+    }
+    if (TYPE === 'selenium') {
+        // Get version of ChromeDriver
+        let chromedriver = ci.sh('chromedriver --version', true); // Returns something like ChromeDriver 80.0.3987.16 (320f6526c1632ad4f205ebce69b99a062ed78647-refs/branch-heads/3987@{#185})
```

Comment:
IIUC, this is already supported and installed in our `circleci-qp:6.4.4-jdk11` docker image because it inherits from `circleci/openjdk:8-stretch-node-browsers`, right? Nice, so all the docker stuff is only for local execution, right?

Reply:
Correct, CircleCI has already everything we need. For local execution Docker isn't needed either. Webdriver.io can use your locally installed Chrome as-is. I think the Docker image might only be relevant for testing against AEMaaCS via CM.